src/metabase/api/permission_graph.clj

```diff
@@ -23,6 +24,14 @@
   [[_ s]]
   (name s))
 
+;; Convert a keyword to string without excluding the namespace.
+;; e.g: :schema/name => "schema/name".
+;; Primarily used for schema-name since schema are allowed to have "/"
+;; and calling (name s) returning a substring after "/".
+(defmethod convert :kw->full-str
```

Comment:
Previously we call `(name schema-name)` and it automatically trim "schema/name" => "name" because it recognizes "schema" as the namespace.

Reply:
Seems like the `:kw->str` method is already only used for schema names. Could you just change that one to call `u/qualified-name` instead of `name`, and avoid adding a new method?